include/feeding/action/DetectAndMoveAboveFood.hpp

```diff
@@ -13,22 +13,13 @@ namespace feeding {
 namespace action {
 
 std::unique_ptr<FoodItem> detectAndMoveAboveFood(
-    const std::shared_ptr<ada::Ada>& ada,
-    const aikido::constraint::dart::CollisionFreePtr& collisionFree,
     const std::shared_ptr<Perception>& perception,
     const std::string& foodName,
-    double heightAboveFood,
-    double horizontalTolerance,
-    double verticalTolerance,
     double rotationTolerance,
-    double tiltTolerance,
-    double planningTimeout,
-    int maxNumTrials,
-    const Eigen::Vector6d& velocityLimits,
-    FeedingDemo* feedingDemo = nullptr,
+    FeedingDemo* feedingDemo,
     double* angleGuess = nullptr,
     int actionOverride = -1);
```

Comment:
Commenting on the actionOverride here. It was in place for study-purposes where we can force the robot to choose an action that was not recommended by SP/SPANet.

In the future, I would be in favor of separating the "Detect" from the "MoveAbove". As the fact that they have been merged by default has made it difficult to run experiments. Then, the override could be handled at a higher level.

Reply:
Agreed on separating Detect and MoveAbove. I'll keep action override as-is for this PR, but when we separate those two we should decide where to best place the override.